cmd/step-ca/main.go

```diff
@@ -179,6 +189,67 @@ intermediate private key.`,
 				return nil
 			},
 		},
+		{
+			Name:  "start",
+			Usage: "Starts step-ca with the (optional) specified configuration",
+			// TODO this should accept an optional config parameter that defaults to ~/.step/config/ca.json
+			// as well as an optional token parameter for connecting to the onboarding flow
+			Action: func(c *cli.Context) error {
+				fmt.Printf("Connecting to onboarding guide...\n\n")
+
+				token := c.Args().Get(0)
+
+				res, err := http.Get("http://localhost:3002/onboarding/" + token)
+				if err != nil {
+					log.Fatal(err)
+				}
+
+				body, err := ioutil.ReadAll(res.Body)
+				if err != nil {
+					log.Fatal(err)
+				}
+
+				configuration := config{}
```

Comment:
I would rename the `config` struct as configuration and the variable as `config`, looks better in my opinion.
```go
var config configuration // or config := configuration{}
```